tests/slirp4netns-no-unmount.sh

```diff
@@ -11,7 +11,7 @@ mkdir /run/foo
 mount -t tmpfs tmpfs /run/foo
 mount --make-rshared /run
 
-unshare -n sleep infinity &
+unshare -r -n sleep infinity &
```

Comment:
Without this the tests (also on Ubuntu) are prolonged by 40s due to the 2 x 40 x 0.5s waiting from `wait_for_network_namespace` and `wait_for_network_device`.

Reply:
This seems to need investigation. I don't think we should need `-r` here.

cc @giuseppe

Reply:
the user should already have enough privileges to run these tests, why does the new user namespace make a difference?

Reply:
I have no idea, unfortunately (no my area of expertise). I only noticed this while running the tests.
I reverted the associated commit for now.
@AkihiroSuda, @giuseppe can you open an issue to track this?
(Asking you because you can give more context than unknowledgeable me.)